app/src/main/java/com/fastaccess/provider/timeline/TimelineProvider.java

```diff
@@ -99,12 +102,10 @@
                             .append(from)
                             .append(" ")
                             .bold(issueEventModel.getRequestedReviewer().getLogin());
-                } else if (event == IssueEventType.closed || event == IssueEventType.reopened) {
+                } else if (event == IssueEventType.closed) {
                     if (issueEventModel.getCommitId() != null) {
```

Comment:
I can try it if you want. I actually fixed a mistake you made 😀.

_Sent from my HTC Nexus 9 using [FastHub Debug](https://play.google.com/store/apps/details?id=com.fastaccess.github)_